Localization/fa/OrchardCore.Settings.po

```diff
@@ -102,7 +102,7 @@ msgstr "تنظیمات فایل‌های اسکریپت و CSS"
 #: OrchardCore.Settings\Views\Settings.Edit.cshtml:42
 msgctxt "OrchardCore.Settings.Views.Settings.Edit"
 msgid "Use resources cache busting"
-msgstr ""
+msgstr "از امکان cache busting منابع استفاده شود"
```

Comment:
How Arabic character could be used in French translation?!!

Reply:
`fa` is Persian, not French.

Reply:
Nah, this is my bad, I saw it from my mobile, ya Farsi contains Arabic characters .. thanks anyway